packages/local_auth/local_auth/windows/CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+cmake_minimum_required(VERSION 3.15)
+set(PROJECT_NAME "local_auth")
+set(CPPWINRT_VERSION "2.0.210806.1")
+project(${PROJECT_NAME} LANGUAGES CXX)
+include(FetchContent)
+
+set(PLUGIN_NAME "${PROJECT_NAME}_plugin")
+
+FetchContent_Declare(nuget
+  URL "https://dist.nuget.org/win-x86-commandline/v6.0.0/nuget.exe"
```

Comment:
Should we consider instead making NuGet a `doctor`-recommended package, and referencing the VS copy instead? /cc @cbracken for thoughts on this.

Reply:
VS doesn't include its own version of NuGet.exe, so it needs to be downloaded somewhere. It would be beneficial to have NuGet.exe inside Flutter's installation, yes, but not sure where. Also, I do not know the work required in Flutter to add an exe as a dependency. IMO, this should not be an error, but only a warning. Also, if we go this route, it would be great if doctor checked if nuget.exe is in the path already, and use that as the correct path, allowing developers to override the nuget exe's version.

Reply:
> VS doesn't include its own version of NuGet.exe, so it needs to be downloaded somewhere.

Ah, I just assumed it was since there's UI for installing packages with it in the IDE. If it's not bundled, this seems like the best approach; we generally don't ship toolchains.

We might want to add a doctor suggestion for works to install it themselves and put it in their path though (so that this wouldn't need to run most of the time).

Reply:
Visual Studio uses "MSBuild.exe /restore", not nuget.exe. AFAIK, VS's UI edits the csproj (in case of C# projects) and calls the restore, basically.
CMake 3.15 does support adding nuget package references, which are restored by msbuild, but there are issues with vcxprojs (C++) and PackageReferences (that is why vcxprojs use a packages.config file), so this doesn't actually work (unfortunately). https://stackoverflow.com/a/61910170/1272111